components/centos/centos-openshift-setup/Vagrantfile

```diff
@@ -35,6 +35,8 @@ end
 
 Vagrant.configure(2) do |config|
   config.vm.box = 'projectatomic/adb'
+  
+  config.servicemanager.services = 'openshift'
```

Comment:
If you see [line no 74 of Vagrantfile](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/centos/centos-openshift-setup/Vagrantfile#L74), we are starting openshift service with extra options viz `IMAGE_NAME`, `IMAGE_TAG` and `DOCKER_REGISTRY` like
```
sudo DOCKER_REGISTRY=#{DOCKER_REGISTRY} IMAGE_TAG=#{IMAGE_TAG} IMAGE_NAME=#{IMAGE_NAME} /usr/bin/sccli openshift
```
In order to pass these extra options, one needs to add following lines in Vagrantfile (in exchange of original lines [73-75 of Vagrantfile](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/centos/centos-openshift-setup/Vagrantfile#L73-75)) like

```
config.servicemanager.openshift_docker_registry = #{DOCKER_REGISTRY}
config.servicemanager.openshift_image_name = #{IMAGE_NAME}
config.servicemanager.openshift_image_tag = #{IMAGE_TAG}
config.servicemanager.services = 'openshift'
```

This way, extra parameters are passed via configuration to [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager/issues) plugin. The plugin reads this configuration and starts the openshift service accordingly. The same info is written in Note section of [docs] (https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/using.rst#using-the-adb-with-host-based-tools-eclipse-and-clis) you referenced.